src/alg/ccf/calc_ccf.jl

```diff
@@ -161,6 +163,9 @@ function project_mask_opt!(projection::A2, λs::A1, plan::PlanT ; shift_factor::
                     mask_lo = λ_min(plan.mask_shape,plan.line_list.λ[m]) * shift_factor
                     mask_hi = λ_max(plan.mask_shape,plan.line_list.λ[m]) * shift_factor
                     mask_weight = plan.line_list.weight[m]
+                    if mask_lo < λsle_cur
```

Comment:
Please add a comment for what this if statement is doing.  
Might also rename p0 something more useful like p_left_edge_of_current_line.